### ISSUE TITLE
Developer with a bad attitude

### ISSUE THREAD

**Issue by drabstrup** [Mountenberg/Covfefetime#4]
Hi,

@pelsekongen and I have had a bad experience with one of your developers. In previous issues he shows a really bad attitude in his answers. This hurts my feelings. Pelsekongen is crushed as well. Pleese fix.

**Comment by pelsekongen** [comment]
I agree completely. Lately the same developer has refused cooperation in hourly fefe-times. He now goes Han Solo all the time. Please fix now (or talk to him)

**Comment by Mountenberg** [comment]
Dearest friends.

Ambushing the developer of your altime favourite site will not solve your current issues. However, for the time being, your local EDB-man has implemented a feature that will not only make you shit you fefe but also make you very happy. Please visit the joke section and let me know what you think.

xxx

**Comment by drabstrup** [comment]
Honorable @Mountenberg,

@pelsekongen and I are terribly sorry that you feel ambushed. We just want what's best for our favourite site and it's community. But you got to admit that you have been too sassy in previous threads. Pelsekongen still can't sleep at night and I have recently started the "ethundredebajere" treatment to heal my wounds. But let's put all this behind us, because we all want to achieve the same thing - a perfect Covfefe world full of jokes.

Give my regards to the talented EDB-man. I was so happy when I saw the new feature that I indeed shat my fefe right on the spot.

My deepest love
Drabstrup
